lib/mesa-lib/BUILD

```diff
@@ -9,10 +9,10 @@ else
   OPTS+=" --enable-egl --enable-osmesa --with-gallium-drivers="
 fi  &&
 
-OPTS+=" --with-egl-platforms=drm,x11" &&
-
-if in_depends $MODULE wayland; then
-  OPTS+=",wayland"
+if module_installed wayland; then
```

Comment:
@stumbles using module_installed wayland is flawed, please don't do that. This does not make it optional for the user when wayland is installed. You can use your proposed logic though, just change that "if" to if in_depends $MODULE wayland, then keep the rest of your changes :)